.bazelrc

```diff
@@ -5,6 +5,7 @@ build --enable_platform_specific_config
 # On all platforms, provide: PYTHON3_BIN_PATH=python
 ###############################################################################
 build --action_env=PATH
+build --local_cpu_resources=4
```

Comment:
Seems this is included by accident for local image building.

Reply:
Yes thanks, I removed it. Somehow it is needed to build ray and not crash my linux machine. Not enough RAM I guess :D